### DISCUSSION TITLE
$ operator is invalid for atomic vectors

### DISCUSSION THREAD

**Discussion by zahrafromhome** [teamalgoritma/community#675] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/82038584/173b372a-994c-4e64-80f0-211582d6c7d5)

saya hendak mengubah tipe data beberapa kolom untuk Data Visualization, namun muncul error message seperti ini

**Comment by jafarhf1** [comment, marked as answer]
Dear @zahrafromhome ,

Boleh tolong dilampirkan data ytrends nya bentuknya seperti apa? Terima kasih

**Comment by kinantytasyao** [comment]
Dear @zahrafromhome, 

Untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah lebih dari 7 hari respon terakhir. Semoga dapat dipahami. Terima kasih.
